pallets/pools/src/tranche.rs

```diff
@@ -225,6 +260,41 @@ where
 		self.accrue(now)?;
 		Ok(self.debt)
 	}
+
+	pub fn create_asset_metadata(
+		&self,
+		decimals: u32,
+		parachain_id: ParachainId,
+		token_name: Vec<u8>,
+		token_symbol: Vec<u8>,
+	) -> AssetMetadata<Balance, CustomMetadata>
+	where
+		Balance: Zero,
+		Currency: Encode,
+		CustomMetadata: Parameter + Member + TypeInfo,
+	{
+		let tranche_id =
+			WeakBoundedVec::<u8, ConstU32<32>>::force_from(self.currency.encode(), None);
+
+		AssetMetadata {
+			decimals,
+			name: token_name,
+			symbol: token_symbol,
+			existential_deposit: Zero::zero(),
+			location: Some(VersionedMultiLocation::V1(MultiLocation {
+				parents: 1,
+				interior: X2(Parachain(parachain_id.into()), GeneralKey(tranche_id)),
+			})),
+			additional: CustomMetadata {
+				mintable: false,
+				permissioned: false,
```

Comment:
@offerijns aren't tranche currencies `permissioned = true`?

Reply:
We also started discussing this here: https://github.com/centrifuge/centrifuge-chain/pull/901#discussion_r954636495

I think @NunoAlexandre makes a good point: my initial reason for saying this should be `false`, is the idea that we will have 3 types of currencies:
1. Tranche tokens
2. Permissioned currencies
3. Non-permissioned currencies

But indeed, we can also say (arguably more logically) there are
1. Permissioned currencies
   - Tranche tokens
   - Non-tranche tokens
2. Non-permissioned currencies

And then in the token `PreConditions`, have a tranche token specific check or a non-tranche token permission check.